app/components/unarchive-bucket-dialog/unarchive-bucket-dialog.html

```diff
@@ -0,0 +1,38 @@
+<md-dialog aria-label="unarchive bucket dialog" class="unarchive-bucket-dialog">
+  <md-dialog-content class="sticky-container">
+
+    <div class="unarchive-bucket-dialog__header" layout="column" layout-align="center center">
+      <ng-md-icon icon="warning"
+        layout="column"
+        layout-align="center center"
+        ng-class="bucket.status == 'live' ? 'unarchive-bucket-dialog__red-warning-icon' : 'unarchive-bucket-dialog__yellow-warning-icon'"
```

Comment:
yea i agree- since the money gets returned on archive, i think we should change the status to draft